OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -181,18 +181,19 @@ public Actor ClosestProc(Actor self, Actor ignore)
 					Location = r.Actor.Location + r.Trait.DeliveryOffset,
 					Actor = r.Actor,
 					Occupancy = self.World.ActorsHavingTrait<Harvester>(h => h.LinkedProc == r.Actor).Count()
-				}).ToDictionary(r => r.Location);
+				}).ToList();
```

Comment:
Think we could use ToLookup here to have the best of both worlds.

Reply:
That sounds smart.